python/cudf/cudf/core/udf/lowering.py

```diff
@@ -11,7 +11,7 @@
 
 from cudf.core.udf.typing import MaskedType, NAType
 
-from . import classes
+from . import api
```

Comment:
Can we make these imports absolute to be consistent with the rest of the code-base?